testgrid/README.md

```diff
@@ -40,9 +40,12 @@ Add something like this to your prowjob:
 
 ```yaml
 annotations:
-  testgrid-dashboards: dashboard-name  # a dashboard defined in config.yaml
-  tab-name: some-short-name            # optionally, a shorter name for the tab. If omitted, just uses the job name.
+  testgrid-dashboards: dashboard-name  # a dashboard defined in config.yaml.
+  testgrid-tab-name: some-short-name   # optionally, a shorter name for the tab. If omitted, just uses the job name.
+  testgrid-alert-email: me@me.com      # optionally, an alert email that will be applied to the tab created in the
```

Comment:
Think this should be okay (though in future I'd like to see this abstracted out into a general 'who owns this stuff' metadata thing). Do you want to note comma-separated list of emails is possible here?

Reply:
In this case I chose to deliberately not mention that that works, giving me leeway to choose to parse it differently in the future.

Reply:
Sweet ^^